scripts/validate_docstrings.py

```diff
@@ -24,6 +24,11 @@
 import inspect
 import importlib
 import doctest
+import tempfile
+from contextlib import contextmanager
+
+from flake8.main.application import Application as Flake8
```

Comment:
I think it complicates things unnecessarily importing objects other than modules. `import contextlib` and `import flake8.main.application` seems like the simplest and best way. And then use accordinly `@contextlib.contextmanager` and `app = flake8.main.application.Application`. This way, just by reading those lines it's obvious what is the module and what is being used, and there is not need to go to the imports and start decoding aliases.